docs/how-to/code/internal-user/rockcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: internal-user
+base: bare
+build-base: ubuntu@24.04
+version: 'latest'
+summary: Add an internal user inside a rock
+description: Add an internal user inside a rock, and run a service with it.
+platforms:
+  amd64:
+
+# [docs:rock-services]
+services:
+  web-server:
+    override: replace
+    startup: enabled
+    command: python3.12 -m http.server --cgi
+    user: myuser 
+# [docs:rock-services-end]
+
+# [docs:rock-parts]
+parts:
+  my-part:
```

Comment:
I'm wary of using the `nil` plugin in examples just because it doesn't need `craftctl default` somewhere in the overrides.

What about using the `dump` plugin and filtering the staged files to just include `cgi-bin`?

Reply:
I actually tried the dump plugin first. Not sure if it's a regression but it failed to copy anything (or it copied them but they weren't staged/primed). I am going to investigate that separately.